Dockerfile

```diff
@@ -132,8 +133,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Darga" \
+          release=${REF} \
```

Comment:
Yeah, it would have to be another arg, unless can you do basic logic in those `${}` blocks.  I don't know what interpolates that...shell?